lib/top/mdx_top.mli

```diff
@@ -29,4 +29,4 @@ val eval : t -> string list -> (string list, string list) result
 (** [eval t p] evaluates the toplevel phrase [p] (possibly spawning on
     mulitple lines) with the configuration value [t]. *)
 
-val in_env : string -> (unit -> unit) -> unit
+val in_env : string -> (unit -> 'a) -> 'a
```

Comment:
What is that change for?

Reply:
That is because I cleaned the "execution in environment" feature, it previously was done at a high level no matter the kind of block, and now it is only done for the kind of blocks that have an environment label, and for the Toplevel blocks I need to match on the result of the execution:
```ocaml
                  match
                    Mdx_top.in_env env (fun () ->
                        eval_test ~block:t ?root c test)
                  with
                  | Ok _ -> ()
                  | Error e ->
                      let output = List.map (fun l -> `Output l) e in
                      if Output.equal test.output output then ()
                      else err_eval ~cmd:test.command e
```